neo/SDK/RPC/RpcClient.cs

```diff
@@ -0,0 +1,178 @@
+        }
+
+        public GetBlockHeader GetBlockHeader(string hashOrIndex)
+        {
+            if (int.TryParse(hashOrIndex, out int index))
+            {
+                return RpcSend<GetBlockHeader>("getblockheader", index, true);
+            }
+            return RpcSend<GetBlockHeader>("getblockheader", hashOrIndex, true);
+        }
+
+        public string GetBlockSysFee(int height)
+        {
+            return RpcSend<string>("getblocksysfee", height);
+        }
+
+        public int GetConnectionCount()
+        {
+            return RpcSend<int>("getconnectioncount");
+        }
+
+        public GetContractState GetContractState(string hash)
+        {
+            return RpcSend<GetContractState>("getcontractstate", hash);
+        }
+
+        public GetPeers GetPeers()
+        {
+            return RpcSend<GetPeers>("getpeers");
+        }
+
+        public string[] GetRawMempool()
+        {
+            return RpcSend<string[]>("getrawmempool");
+        }
+
+        public GetRawMempool GetRawMempoolBoth()
+        {
+            return RpcSend<GetRawMempool>("getrawmempool");
+        }
+
+        public string GetRawTransactionHex(string txid)
+        {
+            return RpcSend<string>("getrawtransaction", txid);
+        }
+
+        public TxJson GetRawTransaction(string txid)
+        {
+            return RpcSend<TxJson>("getrawtransaction", txid);
+        }
+
+        public string GetStorage(string script_hash, string key)
+        {
+            return RpcSend<string>("getstorage", script_hash, key);
+        }
+
+        public uint GetTransactionHeight(string txid)
+        {
+            return RpcSend<uint>("gettransactionheight", txid);
+        }
+
+        public Validator[] GetValidators()
+        {
+            return RpcSend<Validator[]>("getvalidators");
+        }
+
+        public GetVersion GetVersion()
+        {
+            return RpcSend<GetVersion>("getversion");
+        }
+
+        public InvokeRet InvokeFunction(string address, string function, StackJson[] stacks)
+        {
+            return RpcSend<InvokeRet>("invokefunction", address, function, stacks);
+        }
+
+        public InvokeRet InvokeScript(string script)
+        {
+            return RpcSend<InvokeRet>("invokescript", script);
+        }
+
+        public List<Plugin> ListPlugins()
+        {
+            return RpcSend<List<Plugin>>("listplugins");
+        }
+
+        public bool SendRawTransaction(string rawTransaction)
+        {
+            return RpcSend<bool>("sendrawtransaction", rawTransaction);
+        }
+
+        public bool SubmitBlock(string block)
+        {
+            return RpcSend<bool>("submitblock", block);
+        }
+
+        public ValidateAddress ValidateAddress(string address)
+        {
+            return RpcSend<ValidateAddress>("validateaddress");
+        }
+
+        // wait for plugin for neo3.
+        public GetNep5Balances GetNep5Balances(string address)
```

Comment:
Do you think that we should move https://github.com/neo-project/neo-plugins/tree/master/RpcNep5Tracker?

Otherwise, maybe this function could be there.

Reply:
@vncoelho Do you mean moving [https://github.com/neo-project/neo-plugins/tree/master/RpcNep5Tracker](https://github.com/neo-project/neo-plugins/tree/master/RpcNep5Tracker) to neo core?

Reply:
I think this function is necessary for SDK. If you guys think this should be out of neo core, the related plugin should be implemented before the release of NEO 3.0.

Reply:
We can implement NEP5 founctions in SmartContract/NEP5 module. Let's remove this method from RPC module.